src/geo/projection/mercator.js

```diff
@@ -46,11 +47,11 @@ class MercatorTileTransform {
         }
     }
 
-    createInversionMatrix(id: UnwrappedTileID): Float64Array {
-         return this._identity;
+    createInversionMatrix(): Float32Array {
+        return this._identity;
     }
 
-    createTileMatrix(id: UnwrappedTileID): Float64Array {
+    createTileMatrix(id: UnwrappedTileID): Float32Array {
```

Comment:
I think `Float32Array` is conflicting with what's actually returned by this function?